sourcecode.sgml is missing from upstream tarball

--- origsrc/gtkimageview-1.6.3/docs/reference/gtkimageview.sgml	2008-11-20 14:35:13.000000000 -0600
+++ src/gtkimageview-1.6.3/docs/reference/gtkimageview.sgml	2008-11-24 22:05:34.453588500 -0600
@@ -3,7 +3,6 @@
                "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
 <!ENTITY compiling SYSTEM "compiling.sgml">
 <!ENTITY dirstructure SYSTEM "dirstructure.sgml">
-<!ENTITY sourcecode SYSTEM "sourcecode.sgml">
 <!ENTITY releasehistory SYSTEM "releasehistory.sgml">
 ]>
 <book id="index" xmlns:xi="http://www.w3.org/2001/XInclude">
@@ -32,7 +31,6 @@
       </para>
       &compiling;
       &dirstructure;
-      &sourcecode;
       &releasehistory;
     </chapter>
     <chapter id = "acknowledgements">
